apps/feedback/models.py

```diff
@@ -171,9 +171,9 @@ def save(self, *args, **kwargs):
             try:
                 rt = RegisterToken(fbr=self, token=token)
                 rt.save()
-                log.info('Successfully registered token for %s' % rt.user)
+                log.info('Successfully registered token for fbr %s with token %s' % (self, rt))
```

Comment:
We don't need to shorten words to weird abbreviations, so next time, please use the full name. :)